scripts/apps/authoring/metadata/metadata.ts

```diff
@@ -770,13 +770,16 @@ function MetaTermsDirective(metadata, $filter, $timeout, preferencesService, des
 
                         searchObj[scope.uniqueField] = t[scope.uniqueField];
                         if (searchUnique) {
-                            return (t.name.toLowerCase().indexOf(term.toLowerCase()) !== -1 ||
-                                    t[scope.uniqueField].toLowerCase().indexOf(term.toLowerCase()) !== -1) &&
-                                !_.find(scope.item[scope.field], searchObj);
+                            return (t.name.toLowerCase().indexOf(term.toLowerCase()) !== -1
```

Comment:
Extra parentheses aren't needed here, remove them. Use `Array.includes` instead of `Array.indexOf`. Term `toLowerCase` is called a lot. Create a variable like `termLower` and use that instead.

Could you also explain the following lines?

```
|| t[scope.uniqueField].toLowerCase().indexOf(term.toLowerCase()) !== -1)
&& !_.find(scope.item[scope.field], searchObj);
```